docs/howtos/lr_schedule.rst

```diff
@@ -0,0 +1,74 @@
+Managing Parameters and State
+=============================
+
+We will show you how to...
+
+* define a learning rate schedule
+* train a simple model using that schedule
+
+.. code-block:: python
```

Comment:
We are switching to the `testcode` directive instead of `code-block`. This will run the code as part of our tests and make sure everything still compiles.

```suggestion
.. testcode::
```